web-app/uni-todo-app/src/components/TodoListItem.tsx

```diff
@@ -15,12 +15,12 @@ const TodoListItem = ({
     <>
       <div className="flex mb-4 items-center">
         {!todoItem.done && (
-          <p className="w-full text-grey-darkest">{todoItem.name}</p>
+          <p className="w-full text-grey-darkest">{todoItem.name}<p>Created on {new Date(todoItem.createdAt).toLocaleDateString()}</p></p>
```

Comment:
Checkout [Luxon](https://github.com/moment/luxon). It will be better to have it here.
Also please add styling on the new element